rules/editor-required-blazor-component-parameters/rule.md

```diff
@@ -0,0 +1,90 @@
+---
+type: rule
+archivedreason:
+title: Do you use the EditorRequired attribute for required parameters in Blazor?
+guid: 125bf5aa-7b54-4aa3-8f71-8178af185afc
+uri: editor-required-blazor-component-parameters
```

Comment:
```suggestion
uri: editor-required-blazor
```

If you agree, change foldername as well

Reply:
As per convo with @bradystroud, happy to leave as is.